### PR TITLE
Improve kirby-page for desktop usage

### LINKED ISSUE #2733: [Enhancement] Page - desktop ready
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Make the Kirby-page component more useable for desktop solutions. 

## Describe the solution you'd like
The page component should support different content widths and different paddings so that the page could be used both for mobile/tablet and desktop solutions.

**Padding left and right:**
Mobile 16 px.
Tablet 48 px.
Desktop 48 px.

**Padding top:**
Discuss where the top-margin is placed (in header or...)

**Content width:**
Default 720 px.
Standard 792 px.
Optimized 1092 px.
Full Scaled

## Are there any additional context?
Page-spec

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by thupi**
## Which issue does this PR close?

This PR closes #2733 

## What is the new behavior?

This adds a new property called 'maxWidth' that allows the implementer to select a preset that defines the max-width for the content area. It also adds responsive padding so that larger device will get a larger padding.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Are there any additional context?

Some changes were made to local navigation in order to keep it from breaking due to the now dynamic padding size.

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/src/lib/components/page/page.component.scss
Comment (mark-drastrup) [inline libs/designsystem/src/lib/components/page/page.component.scss]: Can we remove this whole block?

**Review by mark-drastrup — DISMISSED** [review]
Looks really great! I just have 1 small comment 😄 Remember to get a review from @henrikvoetmand before merging :)

File: libs/designsystem/src/lib/components/page/page.component.scss
Comment (thupi) [reply libs/designsystem/src/lib/components/page/page.component.scss]: Nooo, missed that one in my self-review 😄 Should be gone from the surface of the earth by now 😃

**Review by mark-drastrup — APPROVED** [review]

**Pull request merged**
